src/Symfony/Component/DependencyInjection/Tests/Fixtures/php/services_inline_requires.php

```diff
@@ -37,10 +37,10 @@ public function __construct()
         $this->aliases = [];
 
         $this->privates['service_container'] = function () {
-            include_once $this->targetDirs[1].'/includes/HotPath/I1.php';
-            include_once $this->targetDirs[1].'/includes/HotPath/P1.php';
-            include_once $this->targetDirs[1].'/includes/HotPath/T1.php';
-            include_once $this->targetDirs[1].'/includes/HotPath/C1.php';
+            include_once \dirname(__DIR__, 0 + 1).'/includes/HotPath/I1.php';
+            include_once \dirname(__DIR__, 0 + 1).'/includes/HotPath/P1.php';
+            include_once \dirname(__DIR__, 0 + 1).'/includes/HotPath/T1.php';
+            include_once \dirname(__DIR__, 0 + 1).'/includes/HotPath/C1.php';
```

Comment:
Wouldn’t it make sense to assign this repeated dormant to a variable?

Reply:
this `dirname` call gets resolved at compile time by OPCache, as it involves only constants. And then this means that the concatenation gets the same. A variable is not worth it.

Reply:
Okidoki 👍